### PR TITLE
Added check for if git exists

### PR DESCRIPTION
**Pull request by geordish**
Should hopefully not stop validate running if git is missing.

DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [X] Have you signed the [Contributors agreement](http://docs.librenms.org/General/Contributing/)
- [X] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

### CODE REVIEW

File: validate.php [hunk 1]
```diff
@@ -75,6 +75,11 @@
     exit;
 }
 
+$git_found = check_git_exists();
+if ($git_found !== true) {
```
Comment (Gorian) [inline validate.php]: could you use `if (!$git_found){` instead?

File: validate.php [hunk 2]
```diff
@@ -103,9 +108,11 @@
     print_fail('You need to run this script as root' . (isset($config['user']) ? ' or '.$config['user'] : ''));
 }
 
-if ($config['update_channel'] == 'master' && $cur_sha != $versions['github']['sha']) {
-    $commit_date = new DateTime('@'.$versions['local_date'], new DateTimeZone(date_default_timezone_get()));
-    print_warn("Your install is out of date, last update: " . $commit_date->format('r'));
+if ($git_found === true) {
```
Comment (Gorian) [inline validate.php]: could you use `if ($git_found){` here?

**Comment by Gorian** [comment]
I think that @mention-bot thing is really cool. Look at who last contributed to a file and then mention them to review new changes.

File: validate.php (hunk 2 above)
Comment (laf) [inline validate.php]: === is better, it's more explicit

Comment (Gorian) [inline validate.php]: Alright, as long as we are doing it purposefully, you are the master of the php code ;)

**Pull request merged**